setup.py

```diff
@@ -29,22 +29,40 @@
 "gfortran" and "python-dev" packages installed.
 '''
 
-import setuptools
-import os
-import sys
+import glob, os, setuptools, shutil, site, sys
 from os.path import join
 
 import nose
 from numpy.distutils.core import setup, Extension
+from setuptools.command.develop import develop
+from setuptools.command.install import install
 
+try:
+    from holopy import __version__
+except ImportError:
+    __version__ = 'unknown'
 HOLOPY_NOSE_PLUGIN_LOCATION = ('holopycatchwarnings = '
                                'holopy.core.tests.common:HoloPyCatchWarnings')
 
 hp_root = os.path.dirname(os.path.realpath(sys.argv[0]))
 
-# this will automatically build the scattering extensions, using the
-# setup.py files located in their subdirectories
-def configuration(parent_package='',top_path=' '):
+class PostDevelopConfig(develop):
+    """Post-installation for development mode."""
+    def run(self):
+        develop.run(self)
+        if os.name == 'nt': _move_S_msvc_libs('develop')
```

Comment:
This function doesn't exist. I think it should just be `_move_msvc_libs`

Reply:
Yeah, I updated this in @5fdef54